tox.ini

```diff
@@ -0,0 +1,53 @@
+[tox]
+downloadcache = {toxworkdir}/_download/
+envlist = py27-dj{14,18},coverage,pep8,pylint,pyflakes
+
+[testenv]
+commands = {envpython} manage.py test
+
+[testenv:py27-dj14]
+deps =
+    django == 1.4.22
+
+[testenv:py27-dj18]
+deps =
+    django >= 1.8, < 1.9
+
+[testenv:pep8]
+deps = pep8
+commands = {envbindir}/pep8 submit_and_compare/
+
+[testenv:pylint]
+deps = pylint
+commands = {envbindir}/pylint submit_and_compare/
+
+[testenv:pyflakes]
+deps = pyflakes
+commands = {envbindir}/pyflakes submit_and_compare/
+
+[testenv:coverage]
+deps =
+    coverage
+
+
+setenv =
+    NOSE_COVER_TESTS=1
+    NOSE_WITH_COVERAGE=1
+
+commands =
+    {envpython} manage.py test
+
+[testenv:coveralls]
+deps =
+    coverage
+    coveralls
+
```

Comment:
TL;DR: Can we remove empty lines from within these `testenv` stanzas?
Newlines _between_ stanzas are fine here.

I think newlines are great for visually grouping blocks of ideas, but like here, not only is this inconsistent (some have no newlines, some have one or more), but I find it actually harder to read.

If I skim through this file, I see a few testenvs logically grouped;
- pylint: lines 20-22
- pep8: 16-18
- etc...

But then `coverage` and `coveralls` are split up fairly arbitrarily so my eye naturally sees it as more than a half dozen groups and it's difficult to tell what goes where without further scrutiny.